node/cn/gasprice/gasprice.go

```diff
@@ -35,14 +35,15 @@ import (
 
 const sampleNumber = 3 // Number of transactions sampled in a block
 
-var maxPrice = big.NewInt(500 * params.Ston)
+var DefaultMaxPrice = big.NewInt(500 * params.Ston)
```

Comment:
Let's remove from here, and have the constant in GetDefaultConfig.

Reply:
@blukat29 Currently, we are using `DefaultMaxPrice` in `NewOracle()` to sanitize GPO config parameter `MaxPrice` if it is not properly set.  If `DefaultMaxPrice` is moved to `node/cn/config.go`. we cannot get that value in `NewOracle()` because of cyclic import. We should either 1) move this constant to `GetDefaultConfig` and not sanitize GPO config in `NewOracle` or 2) keep `DefaultMaxPrice` here.

Reply:
Moved `DefaultMaxPrice` to params package, to remove cyclic import.
See 1ac015e

Reply:
How about removing `DefaultMaxPrice` and leave the constant at `GetDefaultConfig()`? If it is referenced multiple times, we may simply type long such as `GetDefaultConfig().GPO.MaxPrice`.

Reply:
@hyunsooda Then we cannot reference that value in `node/cn/gasprice.go`, for example in `NewOracle()`, because of cyclic import. Do you want me to remove sanitizing `config.maxPrice` in `NewOracle()`?

Reply:
Okay, then the latest commit would be better.